src/App.tsx

```diff
@@ -1,14 +1,57 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const FILTER = {
+    FILTER_ALL: 'all',
+    FILTER_ACTIVE: 'active',
+    FILTER_COMPLETED: 'completed',
+  };
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedFilter, setSelectedFilter] = useState(FILTER.FILTER_ALL);
+  const [inputFilter, setInputFilter] = useState('');
```

Comment:
Initializing state with props can lead to unexpected behavior. Consider using a function to derive the initial state from props if necessary.